src/test/integration/login/login_po.js

```diff
@@ -1,21 +0,0 @@
-// Copyright 2017 The Kubernetes Dashboard Authors.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-export default class LoginPageObject {
```

Comment:
I think this file can stay. Just remove redirect to login page for now. Forcing selenium to work with https should not be that hard.